react-app/src/components/ProposalDetailScreen/ProposalDetailScreenAPI.ts

```diff
@@ -443,3 +452,34 @@ export function useProposalQuery(): {
     fetch: callFetch,
   };
 }
+
+export function useGovParamsQuery(): {
+  requestState: RequestState<GovParams | null>;
+} {
+  const govAPI = useGovAPI();
+  const [requestState, setRequestState] =
+    useState<RequestState<GovParams>>(RequestStateInitial);
+
+  const fetch = useCallback(async () => {
+    setRequestState(RequestStateLoading);
+    try {
+      const params = await govAPI.getAllParams();
+      setRequestState(RequestStateLoaded(params));
+    } catch (err: unknown) {
+      if (err instanceof Error) {
+        setRequestState(RequestStateError(err));
+      } else {
+        throw err;
```

Comment:
should still set request state error here with a custom error of "unknown error"? and maybe log to console?